Calculator/Calculator/Form1.cs

```diff
@@ -0,0 +1,199 @@
+using CalculatorLogic;
+
+namespace Calculator
+{
+    public partial class Form1 : Form
+    {
+        CalculatorLogic.CalculatorLogic calculator = new CalculatorLogic.CalculatorLogic();
+        public Form1()
+        {
+            InitializeComponent();
+        }
+
+        private void PluButtonMouseClick(object sender, MouseEventArgs e)
+        {
+            string[] input = inOutLabel.Text.Split(' ');
+            Array.Reverse(input);
+            double.TryParse(input[0], out double number);
```

Comment:
```suggestion
            double.TryParse(input.Last(), out double number);
```
Зачем разворачивать массив для того, чтобы просто взять последний элемент?